doc/classes/ResourceFormatLoader.xml

```diff
@@ -57,6 +57,7 @@
 			<return type="int" />
 			<param index="0" name="path" type="String" />
 			<description>
+				Should return the unique ID for the resource associated with the given path. If this method is not overridden, an [code].uid[/code] file is generated along with the resource file, containing the unique ID.
```

Comment:
```suggestion
				Should return the unique ID for the resource associated with the given path. If this method is not overridden, a [code].uid[/code] file is generated along with the resource file, containing the unique ID.
```
"u" is pronounced like "ju" here so no leading vowel sound (English 🙃)

Reply:
I'd pronounce this "a dot U I D file" in this case, which also supports using "a" :)

Reply:
Exactly, that's what I mean, and the letter "U" is pronounced "ju"